### ISSUE TITLE
[Cthun] Swallow animation interrupts if it happens during weaken.

### ISSUE THREAD

**Issue by Cabro** [FeenixServerProject/Phoenix_1.12.1_Issue_tracker#5246]
If you get swallowed just before the boss is weakened, so before the animation finishes and you are actually teleported into the stomach, then the following bugs happen:
1 - you are not teleported in stomach, instead you can play normally @ boss.
2 - when weaken is ended, you are magically teleported inside the stomach.
3 - you do not get the stomach debuff.

**Comment by Cabro** [comment]
Why WontFix?

**Comment by Opilol** [comment]
At least give reasoning as to why closing instead of not responding...

**Comment by Cabro** [comment]
Is it because it's hard to test? You need a lot of testers...

**Comment by Cabro** [comment]
Reproduction guide:
You need quite a lot of testers, 5+, or Cthun goes crazy and you can't do anything, being ported 24/7.
You need to bring boss to Phase 2, first of all.

Then you need to kill 1 tentacle in stomach, and then you need to be really fast with communication for this bug to appear. You need someone at "ground level" to say when he's being swallowed, and as soon as you hear him say that, you kill the 2nd tentacle. This has to be done instantly. This has to be done before the one being swallowed is teleported in the stomach.

You'll notice the bug in this case.

As for impact: it can both be abused and detrimental. It happened to me, and i was playing a tank. I was effectively out of the fight, since tanks are needed to tank Giant Claws, no use for them in stomach. One tank cannot possibly cover all the area above, and due to the other bug where they submerge prematurely, this bug can lead to a wipe realllllly fast.

**Comment by bdtps** [comment]
@ Cabro

I have video of the bug youre talking about. https://youtu.be/2PmTSlNy9MQ

1. Vulnerable-phase starts as stomach devouring animation starts. Does not get devoured into stomach.
2. After the vulnerable-phase, I get devoured into the stomach without any animation.
3. I do get debuff from the Digestive Acid.
4. As I exit the stomach--far away from flesh tentacle (stomach tentacles)--they still hits me, and I die. You can see my mouseover on my addon clearly showing I took damage from flesh tentacle and not giant tentacle as presumed.

**Comment by Cabro** [comment]
Wow, you were getting the stomach acid debuff. How? I didn't get it when it happened to me.
